earth_enterprise/src/fusion/rasterfuse/gepackgen.cpp

```diff
@@ -37,7 +37,7 @@ void usage(const std::string &progn, const char *msg = 0, ...) {
 
   fprintf(stderr,
           "\n"
-          "usage: %s [options] {--imagery|--terrain} --config <configfile>"
+          "usage: %s [options] {--imagery|--terrain} --config <configfile> --decimation <decimation_threshhold>"
```

Comment:
Should we mention that the decimation threshold is only valid for terrain?

Reply:
Addressing this in the commit I am about to make